docs/cloud-dev-env.md

```diff
@@ -0,0 +1,26 @@
+# Cloud Development Environments
+
+Altis has experimental built-in support for [GitHub Codespaces](https://github.com/features/codespaces), a Cloud-based development environment, that makes it possible to spin up a complete development environment in seconds within your browser, with a full editor based on Visual Studio Code and a fully-featured environment for your project, without having to install anything on your local machine.
+
+This is useful for many reasons:
+    - Few seconds to bootstrap a full development environment.
```

Comment:
Is that true? Let’s word it less like notes e.g. “Bootstrap a full development environment in seconds”

Reply:
Yes, you get the environment setup in seconds, it just takes a bit more to start the server, but shouldn't block starting to work on the code.

Reply:
```suggestion
    - Bootstrap a full development environment in seconds.
```